### PR TITLE
CON-92, CON-98: Add Avail Consensus

### PR DESCRIPTION
**Pull request by DannyS03**
- Adds doc on Avail's consensus
- May move the comparison to a general location in the future

### CODE REVIEW

File: docs/avail/architecture/avail-consensus.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+commonly referred to as DPoS (delegated proof of stake) or NPoS (nominated proof of stake). 
+Polygon's scaling solutions have adapted these enhanced mechanisms, including Polygon 
+Avail.
+
+Avail uses NPoS with a modification in block verification. The actors involved are 
+still validators and nominators.
+
+Light clients can also contribute to data availability on Avail. Avail's consensus 
+requires that two-thirds plus 1 of the validators reach consensus for validity. An 
+attestation to that consensus is what is then submitted to Ethereum.
```
Comment (prabal-banerjee) [inline docs/avail/architecture/avail-consensus.md:67]: @vthunder Should we separate the validium from consensus discussion. IMO, it is easy to make wrong conclusions if we mention light client and Ethereum here. wdyt?

File: docs/avail/architecture/avail-consensus.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+from an avid staker's point of view. However, the differences lay in the underlying
+consensus mechanisms and how validator selection occurs.
+
+In DPoS, a voting-centric election system determines a fixed number of 
+validators to secure the network. Delegators can delegate their 
+stake against candidate network validators by using it as voting power to back 
+delegates. Delegators often support validators on the highest staked, as higher-staked
+validators have a higher chance of election. The delegates with the most votes 
+become the network's validators and can verify transactions. While using their 
+stake as voting power, they are not subject to consequences via slashing if their
```
Comment (prabal-banerjee) [inline docs/avail/architecture/avail-consensus.md:87]: Not all DPoS are same. In theory atleast, delegators should also be vulnerable to slashing.

File: docs/avail/architecture/avail-consensus.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+Avail is not a DAC, but an actual blockchain network with its consensus 
+mechanism, and has its own set of validator nodes and block producers.
+
+## Proof of Stake
+
+:::caution Current validators
+
+With the initial launch of the Avail testnet, validators will be
+internally ran and maintained by Polygon.
```
Comment (vthunder) [inline docs/avail/architecture/avail-consensus.md:38]: nit: 'operated' instead of 'ran'?

File: docs/avail/architecture/avail-consensus.md (hunk 1 above)
Comment (vthunder) [reply docs/avail/architecture/avail-consensus.md:67]: Yes. The attestation part is not an integral feature of the Avail chain, but rather an additional feature we are building (an attestation bridge) specifically to enable validiums. It should be removed here and only mentioned elsewhere wherever we talk about validiums.

**Review by vthunder — APPROVED** [review]
Looks good other than Prabal's comments.

**Comment by DannyS03** [comment]
thanks @prabal-banerjee @vthunder, feedback addressed

**Pull request merged**